### PR TITLE
library commenter settings frontend

### PR DESCRIPTION
**Pull request by camhashemi**
@cvializ @andrewconner this depends on #18781, hidden behind the admin experiment (maybe it should be the 'keep_comments' exp instead)

screen shot 2015-12-09 at 12 49 14 pm

screen shot 2015-12-09 at 12 49 27 pm

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/libraries/libraryVisibilitySelector.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ angular.module('kifi')
         $scope.orgMemberAccessWrite = $scope.library.orgMemberAccess === 'read_only' ? false : true;
 
         $scope.isUserOrgMember = (profileService.me.orgs.length > 0);
+        $scope.isUserAdmin = profileService.isAdmin();
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/libraries/libraryVisibilitySelector.js]: Using the keep_comments experiment is probably better (as you suggested)

File: kifi-backend/modules/shoebox/angular/src/libraries/manageLibrary.js [hunk 2]
```diff
@@ -319,7 +329,8 @@ angular.module('kifi')
             'description': '',
             'slug': '',
             'visibility': 'published',
-            'orgMemberAccess': 'read_write'
+            'orgMemberAccess': 'read_write',
+            'whoCanComment' : 'collaborator'
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/libraries/manageLibrary.js:345]: Is this the default? @JRuff42 Collaborator vs anyone?

**Comment by andrewconner** [comment]
Implementation looks good, :+1:. The design feels a bit cluttered now, imho. We can improve it later, though.

**Comment by camhashemi** [comment]
thanks @andrewconner. I can work on it now or later (or hand it off to a pro like @cvializ), I didn't want to spend too much time but have some ideas for improvement.

**Comment by andrewconner** [comment]
Cool, yeah don't waste a ton of time if it's behind an experiment.

**Comment by cvializ** [comment]
:+1: awesome!

File: kifi-backend/modules/shoebox/angular/src/libraries/manageLibrary.js (hunk 2 above)
Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/libraries/manageLibrary.js:345]: https://app.asana.com/0/32728208409723/71145589065118

**Comment by camhashemi** [comment]
screen shot 2015-12-09 at 6 26 52 pm
screen shot 2015-12-09 at 6 26 57 pm

**Pull request merged**